oak-lucene/src/test/java/org/apache/jackrabbit/oak/plugins/index/lucene/LucenePropertyIndexTest.java

```diff
@@ -3123,7 +3125,8 @@ private String explain(String query){
     private String explainXpath(String query) throws ParseException {
         String explain = "explain " + query;
         Result result = executeQuery(explain, "xpath", NO_BINDINGS);
-        ResultRow row = Iterables.getOnlyElement(result.getRows());
+        Assert.assertEquals(1, StreamUtils.toStream(result.getRows()).count());
```

Comment:
It's an explain query, so there should only be one row. If for some reason there are more, this might be expensive, no?

Reply:
Yes, it should have only 1 result.

`Iterables.getOnlyElement` validates this as well internally. So that's I have added that check explicitly.